### PR TITLE
APDX-10466 bm sdk update

### PR DESCRIPTION
**Pull request by dimasno1**
Task [link](https://appodeal.atlassian.net/jira/software/c/projects/APDX/boards/352?assignee=712020%3A2822434d-3f6c-4ac3-a5d9-fc0ff280bd39&search=iOS&selectedIssue=APDX-10466)

### CODE REVIEW

File: Obj-C/Podfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
-platform :ios, '12.0'
+platform :ios, '13.0'
 workspace 'BidMachine-iOS-Examples.xcworkspace'
 
 source 'https://github.com/appodeal/CocoaPods.git'
```
Comment (staskochkin) [inline Obj-C/Podfile:4]: Please change the source to `https://github.com/bidmachine/CocoaPods-Specs.git`

File: Swift/Podfile [hunk 2]
```diff
@@ -1,14 +1,14 @@
-platform :ios, '12.0'
+platform :ios, '13.0'
 workspace 'BidMachine-iOS-Examples.xcworkspace'
 
 source 'https://github.com/appodeal/CocoaPods.git'
```
Comment (staskochkin) [inline Swift/Podfile:4]: Please, change source here as well

File: Obj-C/Podfile (hunk 1 above)
Comment (dimasno1) [reply Obj-C/Podfile]: ✅

File: Swift/Podfile (hunk 2 above)
Comment (dimasno1) [reply Swift/Podfile]: ✅

**Review by staskochkin — APPROVED** [review]

**Pull request merged**